### DISCUSSION TITLE
how to make the container fixed when scrolled

### DISCUSSION THREAD

**Discussion by bobwatcherx** [flet-dev/flet#604] [General]
I have a problem how to make a widget container fixed when scrolled ?
is there a solution to make it

this sample design
https://miro.medium.com/max/1400/1*Bt0DtEsCzKdpBCPqwHU1HQ.png

it works the same way as floatingactionbuttonhowever .  what I want is a container and a card that is in the bottom center

**Comment by ndonkoHenri** [comment]
I think you can use a stack with absolute positioning:

https://flet.dev/docs/controls/stack#absolute-positioning-inside-stack

**Comment by FeodorFitsner** [comment]
Here is an example:

```python
import flet as ft

def main(page: ft.Page):
    page.overlay.append(ft.Container(ft.Text("I'm inside a container!"), bgcolor="yellow", padding=20, border_radius=5, bottom=20, left=20, right=20))

    page.scroll = "auto"
    for i in range(0, 100):
        page.controls.append(ft.Text(f"Line {i}"))
    page.update()

ft.app(target=main)
```

![2022-11-21_11h44_33](https://user-images.githubusercontent.com/5041459/203144907-0c0fb38a-acbd-43b9-969e-a567b857518d.gif)

**Reply by ndonkoHenri** [reply]
Way more better approach. 
#theOverlay
